lib/runner.js

```diff
@@ -53,14 +53,25 @@ Runner.prototype.addSerialTest = function (title, cb) {
 };
 
 Runner.prototype.addBeforeHook = function (title, cb) {
-	this.tests.before.push(new Test(title, cb));
+	var test = new Test(title, cb);
+	test.type = 'hook';
```

Comment:
imho `new Test.hook(title, cb)` does not look good. In such case, `Test.hook` is a class (function), which has to be uppercase.